app/webpack/javascripts/modules/external_users/claims/fee_calculator/FeeCalculator.UnitPrice.js

```diff
@@ -207,7 +207,7 @@
           self.displayHelp(context, true)
         })
         .fail(function (response) {
-          if (response.hasOwnProperty('responseJSON') && response.responseJSON.errors[0] !== 'insufficient_data') {
+          if (Object.prototype.hasOwnProperty.call(response, 'responseJSON') && response.responseJSON.errors[0] !== 'insufficient_data') {
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3628/commits/93cbb848cec8a454725c4e9df6154df349d2afca#r538439910)

Reply:
> [ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3628/commits/93cbb848cec8a454725c4e9df6154df349d2afca#r538439910)

As above.